src/components/Header/index.jsx

```diff
@@ -140,7 +140,7 @@ const GordonHeader = ({ onDrawerToggle }) => {
 
   return (
     <section className={styles.gordon_header}>
-      <AppBar className={styles.app_bar} position="static">
+      <AppBar className={styles.app_bar} position="static" color="primary">
```

Comment:
I believe this is unnecessary, since `primary` is the default value of the `color` prop for `AppBar`.

Reply:
Correct, yep, I'll fix that, again, this is still a mix of legacy and new code at this point, we have a document tracking which changes we have finished, and are doing internal code reviews to try to ensure completeness as much as possible.